### DISCUSSION TITLE
How to Set vocoder_name in Python API?

### DISCUSSION THREAD

**Discussion by gene1979** [coqui-ai/TTS#3558] [General Q&A]
Hi Coqui TTS Community,

I'm using Coqui TTS's Python API and noticed that while I can specify model_name, there doesn't seem to be an option to set vocoder_name as we can in the CLI. How can we do this in Python?

Example CLI usage:
`tts --text "口が閉じないほど嬉しかった" --model_name tts_models/ja/kokoro/tacotron2-DDC --vocoder_name vocoder_models/ja/kokoro/hifigan_v1
`
Looking for a Python equivalent. Any suggestions?

Thanks,
Gene

**Comment by gavinv587BC** [comment]
me too ， I  am confused as well.

**Comment by eginhard** [comment]
This is now possible in our fork: https://github.com/idiap/coqui-ai-TTS
